packages/shared/src/server/queues.ts

```diff
@@ -105,27 +120,29 @@ export type TQueueJobTypes = {
     id: string;
     payload: TraceUpsertEventType;
     name: QueueJobs.TraceUpsert;
-    _tracecontext?: TCarrier;
   };
   [QueueName.EvaluationExecution]: {
     timestamp: Date;
     id: string;
     payload: EvalExecutionEventType;
     name: QueueJobs.EvaluationExecution;
-    _tracecontext?: TCarrier;
   };
   [QueueName.BatchExport]: {
     timestamp: Date;
     id: string;
     payload: BatchExportJobType;
     name: QueueJobs.BatchExportJob;
-    _tracecontext?: TCarrier;
```

Comment:
why are these gone?

Reply:
@maxdeichmann bullmq automatically propagates the context for us. Therefore, we do not need them to the event body.